CHANGELOG.md

```diff
@@ -7,10 +7,12 @@ All notable changes to this project will be documented in this file.
 - `propagate_flags` keyword for `UVData.frequency_average` which flags averaged samples if any contributing samples were flagged
 
 ### Changed
+- Updated formatting and added more explicit typing in corr_fits.pyx
+- Updated Nants calculations for speed up.
 - `utils.uvcalibrate` now incorporates many more consistency checks between the uvcal and uvdata object. The new keywords `time_check` and `ant_check` were added to control some of these checks.
 
-### Deprecated
 - `utils.uvcalibrate` will error rather than warn if some of the newly added checks do not pass, including if antenna names do not match between the objects, starting in version 2.2. In addition, the `flag_missing` keyword is deprecated and will be removed in version 2.2.
+### Deprecated
```

Comment:
did you mean to change this? I think the uvcalibrate line should be under Deprecated

Reply:
Also, I think the new metadata only read for MWACorrFITS should be listed in the changelog and maybe added to the tutorial.

Reply:
Hmm this is probably a re-base issue. Thanks for catching it. I'll add the section about metadata only reads and look at the tutorial to try to add that in too.

Reply:
Looks like in the tutorial, there is a list of files which are compatible with metadata only, and a single example of how. I'll update the wordage to reflect that mwa corr fits is under the supported set now.